packages/patternfly-4/react-docs/src/components/navigation/navigation.js

```diff
@@ -57,15 +57,23 @@ class Navigation extends React.Component {
               <img src={logo} alt="PatternFly Logo" />
             </Link>
           </div>
-          <div className={css(styles.search)}>
-            <input
-              className={css(styles.input)}
-              placeholder="Find components, templates,..."
-              type="text"
-              value={searchValue}
-              onChange={this.handleSearchChange}
-            />
-          </div>
+          <form className={[css(styles.search), 'pf-c-form'].join(' ')} onSubmit={event => { event.preventDefault(); return false; }}>
```

Comment:
Could the actual form and TextInput components be used here instead? Would be nice to actually use the components. This was put in before they existed, and never converted over.

Reply:
Ah good call, I was wondering if there was a reason this was raw markup instead of React components. Learning a ton of history in this issue, thx for pointing this out @dmiller9911 - totally agree.